src/api/task/api/dto/task.go

```diff
@@ -1,51 +1,62 @@
 package dto
 
 import (
+	"proyectos/src/api/constants"
+	"proyectos/src/api/errors"
 	"proyectos/src/api/task/domain/model"
 	"time"
 )
 
 // Task dto
 type Task struct {
 	ID             int64     `validate:"gt=0" json:"id"`
-	Name           string    `validate:"required,min=2,max=50" json:"name"`
+	Name           string    `validate:"required,min=2,max=500" json:"name"`
 	Description    string    `validate:"required" json:"description"`
 	StartDate      time.Time `validate:"required" json:"start_date"`
```

Comment:
What does start_date stand for here? If it's the creation time of the task why isn't it autogenerated?